### PR TITLE
XMB add an option to show desktop ui aka WIMP

### PR DESCRIPTION
**Pull request by alfrix**
screenshot_20180817_194001

now with sublabel
screenshot_20180818_182412

@twinaphex @bparker06 @Tatsuya79

### CODE REVIEW

**Comment by Tatsuya79** [comment]
I would add a switch to windowed mode [here](https://github.com/libretro/RetroArch/pull/7088/commits/62ec267375439fe818b48c60abf5bb61cf3ee3b0#diff-925944a1821883c705732954dfb57ba8R609) or the focus won't work on certain OS/drivers while in fullscreen (dektop ui stays in the background on win7 x64 nvidia):

```cpp
      if (settings->bools.ui_companion_toggle || force)
      {
         if (settings->bools.video_fullscreen)
            command_event(CMD_EVENT_FULLSCREEN_TOGGLE, NULL);

         win_handle->qtWindow->activateWindow();
         win_handle->qtWindow->raise();
         video_driver_show_mouse();
         win_handle->qtWindow->show();
```

**Comment by alfrix** [comment]
currently on linux (KDE X11) works as intended if you click on the xmb option
if you press enter it will only bring up qt if it was closed, if you alt-tab and press enter on the option it doesn't switch

File: menu/drivers/xmb.c [hunk 1]
```diff
@@ -2359,6 +2359,9 @@ static uintptr_t xmb_icon_get_id(xmb_handle_t *xmb,
                      return xmb->textures.list[XMB_TEXTURE_RECORD];
                   case MENU_ENUM_LABEL_ONSCREEN_DISPLAY_SETTINGS:
                      return xmb->textures.list[XMB_TEXTURE_OSD];
+#ifdef HAVE_QT
+                  case MENU_ENUM_LABEL_SHOW_WIMP:
```
Comment (bparker06) [inline menu/drivers/xmb.c]: this doesn't make sense... you're returning an xmb texture if the label is "show wimp" ?

Comment (alfrix) [reply menu/drivers/xmb.c]: these are the icons besides the labels...

Comment (bparker06) [reply menu/drivers/xmb.c]: ahh my apologies, didn't realize

**Comment by bparker06** [comment]
If @Tatsuya79 and @twinaphex have tested this and they're ok with it, then I'm ok with it.

**Comment by Tatsuya79** [comment]
Looks good, just a nitpick...
We're calling that QT thing a lot of names but it should be "Desktop Menu" to be completely noob friendly I assume.
"UI" wouldn't be understood by some people perhaps?

**Comment by Tatsuya79** [comment]
Thank you! :+1:

**Pull request merged**